src/test/groovy/net/rubygrapefruit/platform/testfixture/JavaVersion.groovy

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.rubygrapefruit.platform.testfixture
+
+class JavaVersion {
```

Comment:
Do we need all this complexity? Is there an easier way to get the major version of Java?

Reply:
Probably, but this code is copied from Gradle, and works well. It's only test code, too.

BTW, I think "silently do nothing on Java 11+" is not a good production behavior for the feature tested here. The code should emit some warning or fail; something we can actually test.

We could also deprecate this functionality in `native-platform` and remove in a future version.